tests/src/harness/shells.rs

```diff
@@ -55,14 +56,17 @@ export PATH=$PATH:$(dirname "{0}")
             Shell::Zsh => format!(
                 r#"#Autogenerated by pazi integ tests
 set -e
+set -u
```

Comment:
Ah, yeah, I think this should probably be moved to `init_for` for `pazi` specifically.